CHANGELOG.md

```diff
@@ -203,6 +203,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2257](https://github.com/Pycord-Development/pycord/issues/2257))
 - Fixed `AuditLogIterator` not respecting the `after` parameter.
   ([#2295](https://github.com/Pycord-Development/pycord/issues/2295))
+- Fixed `AttributeError` in `client.connect` when user has unstable internet
```

Comment:
Is this right?
```suggestion
- Fixed `AttributeError` when attempting to RESUME gateway connection.
```

Reply:
No, since the error is only raised on the initial connect where if it fails to connect `self.ws` would be `None`, on a resume connect `self.ws` is always not `None`

Reply:
`when user has unstable internet` is still not a good changelog entry

Reply:
```suggestion
- Fixed `AttributeError` when failing to establish initial websocket connection.
```